ddtrace/profiling/scheduler.py

```diff
@@ -63,9 +60,9 @@ def flush(self):
         for exp in self.exporters:
             try:
                 exp.export(events, start, self._last_export)
-            except exporter.ExportError as e:
-                LOG.warning("Unable to export profile: %s. Ignoring.", _traceback.format_exception(e))
-            except Exception:
+            except Exception as e:
+                if e.__class__.__name__ == "ExportError":
```

Comment:
Is the idea here that we want both logs on `ExportError`? Can't we do
```suggestion
                if isinstance(e, exporter.ExportError):
```
here? 🤔

Reply:
OK, I think this is better now.

I originally removed the exporter imports out of ignorance.  I was overly pessimistic about how dependencies were specified in the module.  Instead, the top-level module just specifies its types and leaves specific loading to the caller, so it's safe to bring it back.

I verified that this patch does not erroneously load protobuf if libdd is enabled.